plugins/examples/example_sweeper_plugin/tests/test_example_sweeper_plugin.py

```diff
@@ -16,6 +16,7 @@ def test_launched_jobs(sweep_runner: TSweepRunner) -> None:  # noqa: F811 # type
     sweep = sweep_runner(
         calling_file=None,
         calling_module="hydra.test_utils.a_module",
+        task_function=None,
```

Comment:
is this required? with task_function having a default None and everything here by name I would assume this should just work.

Reply:
Not required but isnt explicit better than implicit ?

Reply:
sure. I was asking to see if other plugins would be broken because of this. (submitit, fairtask).

Reply:
They should not be broken. I can check them and send a PR for them as well - just to be double sure.